tests/integration/test_cloudformation.py

```diff
@@ -308,3 +347,13 @@ def test_nested_stack(self):
         # assert that nested resources have been created
         buckets_after = len(s3.list_buckets()['Buckets'])
         self.assertEqual(buckets_after, buckets_before + 1)
+
+    def test_create_cfn_lambda_without_function_name(self):
+        stack_name = 'stack-%s' % short_uid()
+
+        cloudformation = aws_stack.connect_to_service('cloudformation')
```

Comment:
One more small thing: Can we list the Lambda functions before/after the stack creation, to assert that the function was actually created? Otherwise, it seems that we cannot be certain that the creation of the Lambda was successful.